.github/workflows/test.yml

```
@@ -16,11 +15,13 @@ on:
         required: true
       DEPLOY_AWS_SECRET_ACCESS_KEY:
         required: true
+  push:
+    branches-ignore: [main]
 
 jobs:
   test:
     runs-on: ubuntu-latest
-    environment: development
+    environment: ${{ inputs.environment || development }}
```

actionlint (1.7.7)

undefined variable "development". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)